oat-sa/extension-tao-itemqti
# feat/AUT-76/stylesheet-manager-for-passages

**Related to:** [AUT-76](https://oat-sa.atlassian.net/browse/AUT-76)

**Description:**
- Add a Stylesheet manager for passages and isolate the scope of the CSS rules to the passage.
- The stylesheets attached to a passage should not affect the style of the items the passage is included into

**Changes:**

- Refactor handle functions between taoMediaManager and taoItemQti

:exclamation: Pending Solar Preview, depends on Solar Delivery part :exclamation:

**PR related:**
- https://github.com/oat-sa/extension-tao-mediamanager/pull/404

**How to check:**

- Check Items and Test with passages with stylesheets.
- Check adding passage with custom styles and without it and CSS file. (Without custom styles may have not mainClass on the passage)
- Check previews Terre.

[Kitchen environment on the ticket.](https://oat-sa.atlassian.net/browse/AUT-76?focusedCommentId=190406)
